pypeit/core/wavecal/defs.py

```diff
@@ -1,10 +1,8 @@
 """ Module for arcline definitions
 """
-from __future__ import (print_function, absolute_import, division, unicode_literals)
-
 from astropy.table import Table
 
-
+# TODO: This doesn't belong here.
```

Comment:
Should we move it now?

Reply:
I think I added that comment.  This stuff needs to be moved to and handled by the appropriate spectrograph class, but we should do that in a different PR.